jsapp/js/components/mapSettings.es6

```diff
@@ -227,12 +245,17 @@ class MapSettings extends React.Component {
   render() {
     let asset = this.props.asset,
       geoQuestions = this.state.geoQuestions,
-      activeTab = this.state.activeModalTab;
+      activeTab = this.state.activeModalTab,
+      queryLimit = this.state.mapSettings.querylimit || 5000,
+      queryCount = this.state.queryCount;
 
     var tabs = ['colors'];
 
     if (this.userCan('change_asset', asset)) tabs.unshift('overlays');
-    if (geoQuestions.length > 1) tabs.unshift('geoquestion');
+    if (geoQuestions.length > 1) {
+      tabs.unshift('geoquestion');
+    }
+    if (queryCount > 1000) tabs.unshift('querylimit');
```

Comment:
Please make all three of these `if`s  a single line with `{}` around the statement for cleaner code